Classes/EventListener/ProcessBynderAsset.php

```diff
@@ -1,93 +1,61 @@
 <?php
 
-namespace BeechIt\Bynder\Resource;
+namespace BeechIt\Bynder\EventListener;
 
-/*
- * This source file is proprietary property of Beech.it
- * Date: 21-2-18
- * All code (c) Beech.it all rights reserved
- */
+use BeechIt\Bynder\Resource\BynderDriver;
 use BeechIt\Bynder\Service\BynderService;
 use GuzzleHttp\Exception\ClientException;
 use TYPO3\CMS\Core\Cache\Exception\NoSuchCacheException;
-use TYPO3\CMS\Core\Resource\Driver\DriverInterface;
-use TYPO3\CMS\Core\Resource\File;
 use TYPO3\CMS\Core\Resource\ProcessedFile;
 use TYPO3\CMS\Core\Resource\ProcessedFileRepository;
-use TYPO3\CMS\Core\Resource\Service\FileProcessingService;
-use TYPO3\CMS\Core\SingletonInterface;
 use TYPO3\CMS\Core\Utility\GeneralUtility;
-use TYPO3\CMS\Extbase\Utility\DebuggerUtility;
 
-/**
- * Class AssetProcessing
- *
- * Create/fetch CDN urls for scaled/cropped Bynder assets
- *
- * @package BeechIt\Bynder\Resource
- */
-class AssetProcessing implements SingletonInterface
+class ProcessBynderAsset
 {
-    /**
-     * Timestamp in microseconds
-     *
-     * @var float
-     */
-    protected static $lastRequestedOtfAsset;
-
-    /**
-     * @var BynderService
-     */
-    protected $bynderService;
+    /** @var \BeechIt\Bynder\Service\BynderService */
+    private $bynderService;
 
     /**
-     * @param ProcessedFile $processedFile
-     * @return bool
+     * @param  \BeechIt\Bynder\Service\BynderService  $bynderService
```

Comment:
```suggestion
     * @param  BynderService  $bynderService
```
Shouldn't it be sufficient as you have the use statement above?